compiler/rustc_target/src/spec/targets/armv7_unknown_trusty.rs

```diff
@@ -0,0 +1,37 @@
+use crate::spec::{LinkSelfContainedDefault, PanicStrategy, RelroLevel, Target, TargetOptions};
+
+pub fn target() -> Target {
+    Target {
+        // It's important we use "gnueabi" and not "musleabi" here. LLVM uses it
+        // to determine the calling convention and float ABI, and it doesn't
+        // support the "musleabi" value.
+        llvm_target: "armv7-unknown-unknown-gnueabi".into(),
+        metadata: crate::spec::TargetMetadata {
+            description: Some("Armv7-A Trusty".into()),
+            tier: Some(2),
+            host_tools: Some(false),
+            std: Some(false),
+        },
+        pointer_width: 32,
+        data_layout: "e-m:e-p:32:32-Fi8-i64:64-v128:64:128-a:0:32-n32-S64".into(),
+        arch: "arm".into(),
+        options: TargetOptions {
+            abi: "eabi".into(),
+            features: "+v7,+thumb2,+soft-float,-neon".into(),
```

Comment:
@workingjubilee wrote in this [comment](https://github.com/rust-lang/rust/pull/103895#discussion_r1122465894):

> Slightly surprised to see this target enables Thumb instructions but is not named `thumbv7`.

Reply:
Looking at our targets many of them have `thumb2` activated [without being named `thumbv7`](https://github.com/search?q=repo%3Arust-lang%2Frust+thumb2&type=code).

It seems like our `thumbv7` targets are either based on [`thumbv7` llvm targets](https://github.com/search?q=repo%3Arust-lang%2Frust+llvm_target%3A+thumbv7&type=code) or [`+thumb-mode`](https://github.com/search?q=repo%3Arust-lang%2Frust+%2Bthumb-mode&type=code).

I therefore thinks the naming of this target is consistent with our existing targets.

Reply:
Ah, sure then. I was mostly curious I guess.